lib/internal/process/main_thread_only.js

```diff
@@ -159,18 +157,16 @@ function setupSignalHandlers(internalBinding) {
 }
 
 function setupChildProcessIpcChannel() {
-  // If we were spawned with env NODE_CHANNEL_FD then load that up and
-  // start parsing data from that stream.
-  if (process.env.NODE_CHANNEL_FD) {
-    const fd = parseInt(process.env.NODE_CHANNEL_FD, 10);
-    assert(fd >= 0);
+  const assert = require('assert').ok;
```

Comment:
Nit: `strict` was correct. Otherwise there's no need to access `ok` as it's the same property as the one exported by default.